test/functions/hilde/inlineKeyword.chpl

```diff
@@ -1,5 +1,5 @@
 // Test that inline works as a keyword.
-inline operator real.+(a: real, b: real) return 10.0;
+inline operator +(a: real, b: real) return 10.0;
```

Comment:
Just to make sure I'm understanding, this makes the ambiguity error go away because the + overload in `ChapelBase` is generic and/or standalone where this one is a method and/or non-generic?  I'm having trouble rationalizing why, with this change, this one should win or cause this to change.

(that said, I also don't see that this test is really testing anything useful... it certainly doesn't seem to prove that inline works, as the name would suggest?  I.e., I'm not invested in it and would be fine deleting it, but wouldn't understand the change here if it were something I cared about).

Reply:
IIRC it is just that we now consider isMoreVisible for functions but not for methods. Since this test is trying to define a `+` that is called because it is more visible, it needs to not be a method.

Reply:
You know, I realize now I was reading the diff backwards and thinking the `real` was being added.  I like the new form better.